crates/notion-core/src/tool/mod.rs

```diff
@@ -55,6 +55,19 @@ impl ToolSpec {
         }
         Ok(())
     }
+
+    pub fn uninstall(&self, session: &mut Session, remove_all: bool) -> Fallible<()> {
+        match self {
+            ToolSpec::Node(_version) => unimplemented!("Uninstalling node not supported yet"),
+            ToolSpec::Yarn(_version) => unimplemented!("Uninstalling node not supported yet"),
```

Comment:
Isn't this supposed to uninstall Yarn and Npm?

Reply:
This is only to implement #319, which is uninstall for package binaries. I created a new issue #327 to track implementing uninstall for Node and Yarn.

Reply:
I actually meant the message. I think its fixed now :)